LoRaEngine/LoraKeysManagerFacade/FacadeStartup.cs

```diff
@@ -95,6 +140,16 @@ internal static ConfigHandler Create(IFunctionsHostBuilder builder)
 
             internal abstract string IoTHubConnectionString { get; }
 
+            internal abstract string IoTCentralToken { get; }
```

Comment:
All those are more DPS related than IoT Central central related. So I would rename all to be DeviceProvisioningService rather than IoTCentral. But again, I may miss something specific somewhere.